### ISSUE TITLE
bdd_train = berkely_driving_dataset(PATH_TO_BERKELY_DATASET, transform=bdd_transforms,  type='train', color = False). Why color is false in training script?

### ISSUE THREAD

**Issue by sachinsharma9780** [fabvio/ld-lsi#7]

**Comment by sachinsharma9780** [comment]
I am confused here, why you have passed color=False in training script. Shouldn't it always be color=True since then only it will take ground truth in which lanes are marked as ego and other lanes. What's the role of black ground truth images?

**Comment by fabvio** [comment]
Hi, this is intentional. The images loaded during training are not completely black, instead they have contiguous ID for each class (so 0 for background, 1 for ego lane and 2 for other lanes). By the way, this is the way pytorch requires semantic labels.

If you have additional concerns feel free to reopen the issue 👍
